app/integrations/server/lib/eventArgumentHandler.ts

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Yes apparently, sorry about that. I used this rule because eslint is still complaining to me about it event though the original js code is in snake case
![image](https://user-images.githubusercontent.com/23219524/114244541-1485ed00-9965-11eb-9dba-db9c77f96ddc.png)

Am I missing something? Could you help me solve this?

Reply:
Sorry for late response. Weird that the rule is still being used. Probably we're using an old version of eslint 😬 if it's giving you issues, you can disable the rule `for the block with _` and then enable it after, instead of disabling it for the whole file.

Reply:
Nice idea, thanks